cfpb/consumerfinance.gov
# Remove unneeded js-loader

[In base.py](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/v1/models/base.py#L368-L369), we set `media_js` to a deduped list of files provided as `js` properties of the `Media` class. We then loop through these in `base.html` (along with other page js) and load all the js asynchronously via the `jsl` loader.

However, we somewhat duplicated this behavior in the `youtube-api` and `js-loader` modules, which together, via the `embedVideoScript` method, dedupe and asynchronously add a script to the page.

I added a check in `base.html` to see if `https://` was present in a `media_js` entry and, if so, loads that url as a script directly (instead of trying to resolve a static file). This allows full urls to be provided to the `Media` class, which allowed me to remove the duplicated logic for loading just the youtube iframe API.

## Testing:
 - Pull and build
 - Visit a page with a video, like http://localhost:8000/about-us/blog/addressing-racial-inequities-consumer-finance-markets/
 - You should be able to play the video on the page
